pkg/common/utils/k8s/client.go

```diff
@@ -223,3 +225,70 @@ func GetConfigMaps(ctx context.Context, k8scient client.Client, namespace string
 	}
 	return configMaps, nil
 }
+
+// get the Service by namespace and name.
+func GetService(ctx context.Context, k8sclient client.Client, namespace, name string) (*corev1.Service, error) {
+	var svc corev1.Service
+	if err := k8sclient.Get(ctx, types.NamespacedName{Name: name, Namespace: namespace}, &svc); err != nil {
+		return nil, err
+	}
+	return &svc, nil
+}
+
+func GetPods(ctx context.Context, k8sclient client.Client, namespace string) (corev1.PodList, error) {
+	pods := corev1.PodList{}
+
+	options := client.ListOptions{
+		Namespace: namespace,
+	}
+
+	err := k8sclient.List(ctx, &pods, &options)
+	if err != nil {
+		return pods, err
+	}
+
+	for _, pod := range pods.Items {
+		fmt.Printf("pod --- Name: %s,  pod: %s \n", pod.GetName(), pod.Status.PodIP)
+	}
+	return pods, nil
+}
+
+func GetConfig(ctx context.Context, k8sclient client.Client, configMapInfo *dorisv1.ConfigMapInfo, namespace string, componentType dorisv1.ComponentType) (map[string]interface{}, error) {
```

Comment:
getStartConfig.
add comment, add  comment, add comment.
this function has complicated logic, please add a comment.